client/src/test.ts

```diff
@@ -17,7 +17,9 @@ declare const require: {
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting()
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
+}
```

Comment:
Gosh, you'd think the auto-updater would be smarter about this 😜

```suggestion
  platformBrowserDynamicTesting(),
  {
    teardown: { destroyAfterEach: false }
  }
```